lib/commands/appmanager-livesync.ts

```diff
@@ -8,21 +8,27 @@ export class AppManagerLiveSyncCommand implements ICommand {
 		private $mobileHelper: Mobile.IMobileHelper,
 		private $appManagerService: IAppManagerService,
 		private $errors: IErrors,
-		private $logger:ILogger) { }
+		private $logger:ILogger,
+		private $config: Config.IConfig) { }
 
 	public execute(args: string[]): IFuture<void> {
 		return ((): void => {
+			let windowsPhonePlatformName = this.$mobileHelper.normalizePlatformName("WP8");
```

Comment:
My idea was to make sure that even if we change the value of  WP8 in mobileHelper.platforms, this code will still work. In case I put "WP8" here and later we decide to change the value in mobileHelper.platforms, we'll fail here.